packages/framework/src/Component/Router/FriendlyUrl/FriendlyUrlRepository.php

```diff
@@ -137,6 +137,29 @@ public function getAllByRouteNameAndEntityId($routeName, $entityId)
         );
     }
 
+    /**
+     * @param string $routeName
+     * @param int $entityId
+     * @param int[] $domainIds
+     * @return \Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\FriendlyUrl[]
+     */
+    public function getAllByRouteNameDomainIdsAndEntityIds(string $routeName, int $entityId, array $domainIds): array
+    {
+        $criteria = [
+            'routeName' => $routeName,
+            'entityId' => $entityId,
+            'domainId' => $domainIds,
```

Comment:
this works? Doctrine is so clever that it knows it needs to use `IN` here? Nice :slightly_smiling_face:

Reply:
yes, yes it is :)